app/models/user.rb

```diff
@@ -0,0 +1,27 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :posts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
+           foreign_key: "follower_id",
+           dependent: :destroy
+  has_many :passive_relationships, class_name: "Relationship",
+           foreign_key: "followed_id",
+           dependent: :destroy
+
+  validates :profile, length: { maximum: 200 }
+
+  # ユーザーが他のユーザーをフォローするメソッド
+  def follow(followed_id)
+    active_relationships.create(followed_id: followed_id)
+  end
+  # ユーザーが他のユーザーのフォローを解除するメソッド
+  def unfollow(followed_id)
```

Comment:
例外を発生させるメソッドを呼ぶ（destroy!）メソッドの名前も ! を付けます。
```suggestion
  def unfollow!(followed_id)
```